core/templates/dev/head/pages/exploration_player/ExplorationPlayer.ts

```diff
@@ -62,3 +62,19 @@ require('pages/exploration_player/ConversationSkinDirective.ts');
 require('pages/exploration_player/ExplorationFooterDirective.ts');
 require('pages/exploration_player/LearnerLocalNav.ts');
 require('pages/exploration_player/LearnerViewInfo.ts');
+
+require('services/ContextService.ts');
+require('services/PageTitleService.ts');
+require('domain/exploration/ReadOnlyExplorationBackendApiService.ts');
+
+oppia.controller('ExplorationPlayer', ['ContextService', 'PageTitleService',
+  'ReadOnlyExplorationBackendApiService',
+  function(ContextService, PageTitleService,
+      ReadOnlyExplorationBackendApiService) {
+    var explorationId = ContextService.getExplorationId();
+    ReadOnlyExplorationBackendApiService.fetchExploration(explorationId,
```

Comment:
Break after `(`

Reply:
Also, do we need to fetch the whole exploration here? Isn't there some better way to get the title?

Reply:
Not really.
Fetching the whole exploration also comes with values like can_edit, is_admin, iframed, etc which can be used to remove GLOBALS in the exploration_player.html